app/src/main/res/values/arrays.xml

```diff
@@ -28,10 +28,10 @@
     </string-array>
     <string-array name="remote_languages">
         <item>English</item>
-        <!--        <item>Spanish (beta)</item>-->
+        <item>Spanish (beta)</item>
     </string-array>
     <string-array name="remote_languages_values">
         <item>en</item>
-        <!--        <item>es</item>-->
+        <item>es</item>
```

Comment:
It seems that the API in Spanish is no longer available, so we should remove this option from the configuration and remote recipe search

Reply:
I have commented this line again. In order to remove Spanish from available remote languages, we should create a new issue.